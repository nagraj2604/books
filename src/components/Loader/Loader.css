.loader {
  padding: 4rem 0;
}
.loader img {
  width: 7.5rem;
}
