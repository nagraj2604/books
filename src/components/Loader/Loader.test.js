import React from "react";
import { render, cleanup } from "@testing-library/react";
import Loader from "./Loader";

afterEach(cleanup);

it("renders", () => {
  const { asFragment } = render(<Loader />);
  expect(asFragment()).toMatchSnapshot();
});
