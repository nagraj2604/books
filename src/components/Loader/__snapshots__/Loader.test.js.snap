// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="loader flex flex-c"
  >
    <img
      alt="loader"
      src="loader.svg"
    />
  </div>
</DocumentFragment>
`;
